src/app/docs/components/search/search.tsx

```
@@ -79,20 +79,20 @@
-                ? (
-                    results.map((result, index) => <Result key={index} result={result} />)
-                  )
-                : (
-                    <div className={cx('no-results')}>
-                      {translations[lang].search.noResults.replace('{{query}}', query)}
-                    </div>
-                  )}
-            </div>,
-            resultsRef.current!,
-          )}
+        && createPortal(
```

ESLint

[error] Expected indentation of 10 spaces

(@stylistic/indent-binary-ops)